OurUmbraco.Site/Views/Partials/Projects/ListProjects.cshtml

```diff
@@ -185,7 +185,7 @@
                                 defaultScreenshot = childContent.GetPropertyValue("defaultScreenshotPath", false, "/css/img/package2.png");
                             }
                         }
-                        <img src="@Utils.GetScreenshotPath(defaultScreenshot)?bgcolor=fff&width=50&height=50&format=png" />
+                        <img src="@Utils.GetScreenshotPath(defaultScreenshot)?bgcolor=fff&width=100&height=100&format=png" />
                     </div>
```

Comment:
Not sure how the stats for Our in either, but I guess it is most developers. So I am not sure how many of the visitors, who actually use IE and Edge.
As always IE and Edge lack it support on newer technology compared to other browsers: http://caniuse.com/#search=srcset

However it does seem that Edge is supporting srcset in the latest versions: https://blogs.windows.com/msedgedev/2015/06/08/introducing-srcset-responsive-images-in-microsoft-edge/#XXSF2GfTD6HgteM7.97